tests/aws/services/cloudformation/api/test_stacks.py

```diff
@@ -642,3 +651,94 @@ def test_blocked_stack_deletion(aws_client, cleanups, snapshot):
     snapshot.match("stack_post_success_delete", stack_post_success_delete)
     stack_events = cfn.describe_stack_events(StackName=stack_id)
     snapshot.match("stack_events", stack_events)
+
+
+MINIMAL_TEMPLATE = """
+Resources:
+    SimpleParam:
+        Type: AWS::SSM::Parameter
+        Properties:
+            Value: test
+            Type: String
+"""
+
+MINIMAL_TEMPLATE_UPDATED = """
+Resources:
+    SimpleParam:
+        Type: AWS::SSM::Parameter
+        Properties:
+            Value: test2
+            Type: String
+"""
+
+
```

Comment:
nit: this template is not used

Reply:
Yep  good catch. Was going through a few iterations of the test cases, one also had an update with a different template after a successful create. I decided against this to keep the already kinda long test from bloating too much.